### PR TITLE
Local packer

### PR DESCRIPTION
**Pull request by heckj**
- updating packer readme for local builds and notes on accessing vagrant boxes locally 
- workaround for https://github.com/RackHD/RackHD/issues/357

### CODE REVIEW

**Comment by anhou** [comment]
:+1:

File: packer/ansible/roles/remoterepos/tasks/main.yml [hunk 1]
```diff
@@ -69,6 +69,12 @@
   args:
     chdir: "{{ ansible_env.HOME }}/src/on-http"
 
+#workaround for https://github.com/RackHD/RackHD/issues/357
+- name: install dev dependencies for on-http to allow taskdoc to work
+  shell: "npm install"
```
Comment (yyscamper) [inline packer/ansible/roles/remoterepos/tasks/main.yml]: @WangWinson has submitted the PR (https://github.com/RackHD/on-tasks/pull/303) to not rely on dev dependencies, so I don't think we need this.

**Comment by benbp** [comment]
👍

**Comment by heckj** [comment]
@anhou @benbp rebased out the workaround for the taskdoc rendering

**Pull request merged**